README.md

```diff
@@ -1,8 +1,48 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Awais Zafar . Jarvis Consulting
 
+I have the ability to break things down and communicate to any level. I have the right mix of strategic thinking and the ability to influence gained over the years working as an sole propritor. I am extremely good at building up and nurturing interpersonal relationships. I have a high awareness of business goals and business organization. I am very attentive to details and am always willing to take the extra step to get things accomplished. I also have utilized requirement elicitation skills in the previous job to understand all the client's needs and objectives. I am very good at making sure things get done and get done right. I have fine people skills, am analytical, and can think on my feet combined with savvy, computer skills. I have utilized the skills needed for presenting on several occasions academically and professionally . As an investor in Forex and Stocks, I have the basic knowledge of how wealth management works. Because of my graduate degree in Electronics and Communications Engineering from the University of Kent, England I am well versed in a good range of programming and database skills across many platforms. Therefore, I wish to be considered for a Business System Analyst role where I can use my skills and work in a team setting so I can also work on my leadership skills.
```

Comment:
I have the ability to break things down and communicate to any level. I have the right mix of strategic thinking and the ability to influence gained over the years working as a sole proprietor. I am extremely good at building up and nurturing interpersonal relationships. I have a high awareness of business goals and business organization. I am very attentive to details and always willing to take the extra step to accomplish things. I also have utilized requirement elicitation skills in the previous job to understand all the client's needs and objectives. I am very good at ensuring things get done and done right. I have fine people skills, am analytical, and can think on my feet, combined with savvy computer skills. I have utilized the skills needed for presenting on several occasions academically and professionally. As an investor in Forex and Stocks, I have the basic knowledge of how wealth management works. Because of my graduate degree in Electronics and Communications Engineering from the University of Kent, England, I am well versed in a good range of programming and database skills across many platforms. Therefore, I wish to be considered for a Business System Analyst role where I can use my skills and work in a team setting to improve my leadership skills.